docs/development.md

```diff
@@ -114,9 +114,12 @@ If it is not a valid version try, make sure you have updated the godep repo with
 
 ```console
 cd $GOPATH/src/github.com/skippbox
-godep restore
+script/godep-restore.sh
 ```
 
+Reason for calling `script/godep-restore.sh` instead of just `godep restore` is that Kompose is using `github.com/openshift/kubernetes` instead of `github.com/kubernetes/kubernetes`.
```

Comment:
OpenShift team usually rebase `openshift/kubernetes` as soon as there is a new release of kubernetes.
But I believe that in future we will be able to switch back to `kubernetes/kubernetes` as changes that openshift depends on are going to be merged upstream.